src/usr/local/www/system_advanced_network.php

```diff
@@ -186,6 +199,15 @@
 ))->setHelp('By default, if IPv6 is configured and a hostname resolves IPv6 and IPv4 addresses, '. 
 	'IPv6 will be used. If this option is selected, IPv4 will be preferred over IPv6.');
 
+$section->addInput(new Form_Input(
+	'global-v6duid',
+	'DHCP6 DUID',
+	'text',
+	$pconfig['global-v6duid'],
+	['placeholder' => 'xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx']
+	))->setWidth(9)->sethelp('Enter the DUID to use here. Use this option if using RAM Disk, as the DUID will be lost on reboot. The existing DUID may be found in var/db/dhcp6_duid.' .'<br />' .
+			'NOTE: Do not use this option with multiple DHCP6 WAN interfaces.');
```

Comment:
I only have one WAN interface so I cannot be sure about this, and I know what we said about DUID the other day, but the RFC does say that DUID's should be interface unique. When running dhcp6c on multiple WAN interfaces I assume multiple dhcp6c clients are launched, or is that not the case?

If it is the case then the solution is to go back to the by WAN setting of DUID's and I'll modify dhcp6c to use specific DUIDs for each interface. We already pass an the argument to dhcp6c anyway so it would be very simple to do.